index.html

```diff
@@ -63,86 +70,100 @@ <h1>Eat Donuts in the Wilderness</h1>
+              <p class="card-body">Lots and lots of text about how great donuts are when you eat them by the sea. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut congue a quam in gravida. Ut scelerisque at est consectetur tincidunt. Sed volutpat massa non arcu imperdiet consectetur.</p>
+            </div>
           </div>
-        </div>
-        <div class="card">
-          <img src="./assets/images/mountains.jpg" class="card-image" alt="...">
-          <div class="card-main">
-            <h5 class="card-heading">Donuts in the Mountains</h5>
-            <p class="card-body">And some wonderful reasons why you eat them in the mountains. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut congue a quam in gravida. Ut scelerisque at est consectetur tincidunt. Sed volutpat massa non arcu imperdiet consectetur.</p>
+          <div class="card">
+            <img src="./assets/images/mountains.jpg" class="card-image card-img-top" alt="...">
+            <div class="card-body">
+              <h5 class="card-title">Donuts in the Mountains</h5>
+              <p class="card-text">And some wonderful reasons why you eat them in the mountains. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut congue a quam in gravida. Ut scelerisque at est consectetur tincidunt. Sed volutpat massa non arcu imperdiet consectetur.</p>
+            </div>
           </div>
-        </div>
-        <div class="card">
-          <img src="./assets/images/waterfall.jpg" class="card-image" alt="...">
-          <div class="card-main">
-            <h5 class="card-heading">Donuts near a Waterfall</h5>
-            <p class="card-body">Or by a waterfall. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut congue a quam in gravida. Ut scelerisque at est consectetur tincidunt. Sed volutpat massa non arcu imperdiet consectetur.</p>
+          <div class="card">
+            <img src="./assets/images/waterfall.jpg" class="card-image card-img-top" alt="...">
+            <div class="card-body">
+              <h5 class="card-title">Donuts near a Waterfall</h5>
+              <p class="card-text">Or by a waterfall. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut congue a quam in gravida. Ut scelerisque at est consectetur tincidunt. Sed volutpat massa non arcu imperdiet consectetur.</p>
+            </div>
           </div>
-        </div>
-        <div class="card">
-          <img src="./assets/images/sand.jpg" class="card-image" alt="...">
-          <div class="card-main">
-            <h5 class="card-heading">Donuts in the Dunes</h5>
-            <p>Too hot don't do it unless it's at night!!</p>
+          <div class="card">
+            <img src="./assets/images/sand.jpg" class="card-image card-img-top" alt="...">
+            <div class="card-body">
+              <h5 class="card-title">Donuts in the Dunes</h5>
+              <p class="card-text">Too hot don't do it unless it's at night!!</p>
+            </div>
```

Comment:
This is personal preference on my part but since the image on your card has a "border" look on the left and right I would also put a margin at the top of the card so the look is consistent all the way around. And since the top corners are rounded I would also consider rounding the bottom. The classes on the card image would then look like `class = "card-image card-img-top mt-2 rounded"`. In the image the top margin doesn't quite match the sides so more tweaking would be necessary. ![image](https://user-images.githubusercontent.com/56650147/135560784-f3a38189-15bf-4c7e-9c05-bd852e7dbce0.png)